src/views/SchemaForm.vue

```diff
@@ -164,6 +165,7 @@ export default {
       addField(field) {
           const hasEnum = field.constraints && field.constraints.enum
           const isBoolean = field.type === "boolean"
+          const isAddress = field.name.includes("adresse")
```

Comment:
Can we add possibility for the field name to contain two more possibilities : 
- ```adress```
- ```ad_```

Reply:
"addr" est aussi parfois utilisé. (petite typo sur  "address" qui a 2 "d")

Reply:
Tout à fait :) 
Sinon ok pour ```addr``` ! Merci

Reply:
Est-ce que je résume bien si je dis que la widget devrait être activée si le nom du champ contient un de ces 3 motifs ?
- ad_
- addr
- adresse

Reply:
Juste `addr` ça me parait dangereux. Peut-être `addr_` et `address`?

Reply:
Ok avec la remarque d'@abulte et tant qu'à faire, ajoutons ```adr_```

Reply:
Donc, on retient les sous-chaînes suivantes (non sensible à la casse)  ?
- addr_
- address
- adr_
- adresse

Reply:
il manque ```ad_``` mais sinon la liste est ok.
Non sensible à la casse effectivement ! Merci

Reply:
La liste a été prise en compte, cf. https://github.com/etalab/csv-gg/pull/61/commits/6a4637dce6ee328fc132a1283529c2d27bb0a091